search-provider/eks-discovery-feed-provider.c

```diff
@@ -195,12 +195,20 @@ add_key_value_int_to_str_pair_from_model_to_variant (EkncContentObjectModel *mod
 }
 
 static void
-add_author_from_model_to_variant (EkncArticleObjectModel *model,
-                                  GVariantBuilder        *builder,
-                                  const char             *key)
+add_first_string_value_from_model_to_variant (EkncContentObjectModel *model,
+                                              GVariantBuilder        *builder,
+                                              const char             *model_key,
+                                              const char             *key)
 {
-  char * const *authors = eknc_article_object_model_get_authors (model);
-  add_key_value_pair_to_variant (builder, key, authors ? authors[0] : "");
+  g_auto(GStrv) values;
+  g_autofree gchar *underscore_key = underscorify (key);
+
+  g_object_get (model, model_key, &values, NULL);
+
+  if (values && values[0] != NULL)
+    add_key_value_pair_to_variant (builder, underscore_key, g_strdup (values[0]));
```

Comment:
I don't think it's necessary to copy the string here, `g_variant_builder_add()` will do it already.

Reply:
Ah right. I'll drop the `strdup` then